src/Console/Command/WatchCommand.php

```diff
@@ -67,6 +87,9 @@ protected function execute(InputInterface $input, OutputInterface $output): int
     {
         $console = new OutputStyle($input, $output);
 
+        $runLimit = InputHelper::nullablePositiveInt($input->getOption('run-limit'));
+        $memoryLimit = InputHelper::nullableString($input->getOption('memory-limit'));
+        $timeLimit = InputHelper::nullablePositiveInt($input->getOption('time-limit'));
```

Comment:
I'm not sure if these options are really useful for the watcher, as the current usecase is only in dev. There i dont think you really want to stop the worker e.g. after 10 minutes or so. On the other hand it is not really harmful either and the maintenance cost should be low, wdyt @DavidBadura

Reply:
I also think it doesn't make sense to set any limits in general, since it's just a dev command. It doesn't hurt to maintain it, but it doesn't add any value either.

I've only found one case where you initially think it makes sense, but then you quickly realize that it only provides a false sense of security:

That case is when the limit is exceeded and the command is shut down cleanly and restarted by something else (like Docker or Superviser). However, there's the problem that events are thrown between shutdown and restart, which are then not displayed in the console. In other words, you "lose" events when displaying them, which I find very confusing.

I would tend not to add the limits, but I'm fine with adding them.

Reply:
@llaville Do you perhaps have a use case that I have overlooked?

Reply:
@DavidBadura This is for test case that doesn't exists yet on WatchCommand I've introduced : see https://github.com/llaville/event-sourcing/commit/143c2e659e715b271108bd1beaa01eba25dfc00d#diff-d2abab7733f01348ecec5ddc13ecbe1cb84128e85be13db41fcea44cadfca9ecR42 (line 42)